requirements.txt

```diff
@@ -6,4 +6,4 @@ flasgger==0.9.7.1
 # Add the fertiscan-pipeline package from GitHub
 git+https://github.com/ai-cfia/fertiscan-pipeline.git@main
 # Add the datastore to interact with the database
-git+https://github.com/ai-cfia/ailab-datastore.git@main
+git+https://github.com/ai-cfia/ailab-datastore.git@90-update-a-fertiscan-inspection
```

Comment:
If it's almost done, it might be better to stay on main and merge this after that branch is merged.

Reply:
Done.